src/AbstractSerializer.php

```diff
@@ -61,8 +61,7 @@ public function getRelationship($model, $name)
             $relationship = $this->$name($model);
 
             if ($relationship !== null && ! ($relationship instanceof Relationship)) {
-                throw new LogicException('Relationship method must return null or an instance of '
-                    .Relationship::class);
+                throw new LogicException('Relationship method must return null or an instance of Relationship');
```

Comment:
This change is incorrect anyway. The exception message before and after is NOT the same.